packages/next/build/webpack/plugins/font-stylesheet-gathering-plugin.ts

```diff
@@ -132,19 +137,42 @@ export class FontStylesheetGatheringPlugin {
 
           this.manifestContent = []
```

Comment:
Can't this hook be removed and moved to the new processAsset state 🤔 Doesn't seem like there's anything particular about this hook

Reply:
The font manifest needs to be created in this hook as it's used in the `requireExtension` hook above. Emit would be too late.

Reply:
Okay!